aiida/cmdline/utils/multi_line_input.py

```diff
@@ -31,12 +32,38 @@ def edit_pre_post(pre=None, post=None, summary=None):
     template = env.get_template('prepost.bash.tpl')
     summary = summary or {}
     summary = {k: v for k, v in summary.items() if v}
-    content = template.render(default_pre=pre or '', default_post=post or '', summary=summary)
+
+    # Define a separator that will be splitting pre- and post- execution
+    # parts of the submission script
+    separator = "#====================================================#\n" \
+                "#=               Post execution script              =#\n" \
+                "#=  I am acting a as separator, do not modify me!!! =#\n" \
```

Comment:
you can add the article `a` but then it should come after `as`, i.e. it should be `I am acting as a separator`

Reply:
of course, sorry :)